js-runner/src/core/grain-module.js

```diff
@@ -248,7 +248,8 @@ export async function readFile(filepath) {
 }
 
 export async function readURL(url) {
-  let modname = url; // FIXME
+  // TODO: 
```

Comment:
@ospencer can you provide some context as to what needs to be done here.

Reply:
The url isn't the name of the module, we should use the real name of the module.